#Used for SATO guidance to figure out what other controllers exist/order them
string name                         #the unique name of the controller
geometry_msgs/Point initial_position   #the position of this controller before SATO runs
